src/System.IO.FileSystem.AccessControl/src/System/IO/FileSystemAclExtensions.cs

```diff
@@ -10,6 +10,29 @@ namespace System.IO
 {
     public static class FileSystemAclExtensions
     {
+        /// <summary>
+        /// Creates a new file stream, ensuring it is created with the specified properties and security settings.
+        /// </summary>
+        /// <param name="fileInfo">The current instance describing a file that does not exist in disk yet.</param>
+        /// <param name="mode">One of the enumeration values that specifies how the operating system should open a file.</param>
+        /// <param name="rights">One of the enumeration values that defines the access rights to use when creating access and audit rules.</param>
+        /// <param name="share">One of the enumeration values for controlling the kind of access other FileStream objects can have to the same file.</param>
+        /// <param name="bufferSize">The number of bytes buffered for reads and writes to the file.</param>
+        /// <param name="options">One of the enumeration values that describes how to create or overwrite the file.</param>
+        /// <param name="fileSecurity">An object that determines the access control and audit security for the file.</param>
+        /// <returns>A file stream for the newly created file.</returns>
+        /// <exception cref="ArgumentNullException"><paramref name="fileInfo" /> or <paramref name="fileSecurity" /> is <see langword="null" />.</exception>
+        public static FileStream Create(this FileInfo fileInfo, FileMode mode, FileSystemRights rights, FileShare share, int bufferSize, FileOptions options, FileSecurity fileSecurity)
+        {
+            if (fileInfo == null)
+                throw new ArgumentNullException(nameof(fileInfo));
+
+            if (fileSecurity == null)
+                throw new ArgumentNullException(nameof(fileSecurity));
+
+            return null;
```

Comment:
We should avoid submitting PRs when they aren't functional, aren't meant for review, etc.

Reply:
Yes, apologies. I meant to submit a draft but forgot to select the button when submitting.

Reply:
No need to apologize. But why submit any pr, even a draft? If no one is supposed to review it, and it doesn't contain anything meaningful to review, what's the benefit of opening a pull request?

Reply:
I noticed it has been easier for me to track my commits with a draft PR and have Jeremy help review them. But I have to agree, this empty PR just adds noise. I'll close it and reopen it when I have something meaningful to have reviewed.

Reply:
Thanks